flink-runtime-web/web-dashboard/app/scripts/modules/submit/submit.ctrl.coffee

```diff
@@ -167,7 +167,7 @@ angular.module('flinkApp')
             $scope.uploader['success'] = null
           else
             $scope.uploader['success'] = "Uploaded!"
-      xhr.open("POST", "/jars/upload")
+      xhr.open("POST", "jars/upload")
```

Comment:
Thx, @sachingoel0101 could you provide me some more details on why that is preferrable? Happy to change it, just want to understand the details it :-).